localMatchApp/Sources/logIn.cpp

```diff
@@ -0,0 +1,68 @@
+#include "Headers/logIn.h"
+
+using namespace std;
+
+logIn::logIn()
+{
+    //Connect to database.
+}
+
+logIn::~logIn()
+{
+    //Disconnect to database.
+}
+
+int logIn::signUp(const string i_user, const string i_password, const string i_repeatedPassword)
+{
+  //Send information to database.
+  string userInDatabase="";
```

Comment:
it is more clear to initialize it like:
```c
string userInDatabase = "";
```

apply this same comment in all the places